### PR TITLE
Update contact-pstn-service-desk.md

### PR DESCRIPTION
**Pull request by ErsancMS**
Latest support information from the PSTN Service Desk Teams from ptn@microsoft.com, ptneu@microsoft.com and gcstnmsd@microsoft.com

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 1]
```diff
@@ -24,14 +24,24 @@ ms.custom:
 ROBOTS: NOINDEX, NOFOLLOW
 description: "When you are getting phone numbers or porting (transferring) numbers for your organization, you may need to get help and support at the PSTN service desk."
 ---
-# PSTN service desk help
+# PSTN Service Desk Contact
 
-If you need help with getting or transferring numbers to us but need help please let us know how we can help you in an email and send it to us. In your email Help request, include as many details as you can and include details such as your organization ID, domains, what types of numbers you need help with, how many numbers you want, authorizing person on the account and other important details about your issue.
+If you need help with acquiring/getting or transferring/porting telephone numbers to us but need assistance please let us know how we can help you by sending us an email.
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
If you need help with acquiring/getting or transferring/porting telephone numbers to us but need assistance, please let us know how we can help you by sending us an email.
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 2]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
 
-Then send your request to one of the following email addresses:
+In your email request, please include as many details as you can such as:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
In your email request, please include as many details as you can, such as:
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 3]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+  - Your Organization ID (Tenant ID) or active domains
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
  - Your Organization ID (Tenant ID) or active domains.
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 4]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+  - What types of numbers you need help with (User/Subscriber, Service/Toll or Toll-Free)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
  - What types of numbers you need help with (User/Subscriber, Service/Toll, or Toll-Free).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 5]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+  - How many numbers you require (if this is for number acquisition)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
  - How many numbers you require (if this is for number acquisition).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 6]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+  - Authorizing person on the account and other important details based on your request
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
  - Authorizing person on the account and other important details based on your request.
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 7]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
 
-- For help with phone numbers inside the U.S., [send your request](mailto:ptn@microsoft.com).
-- For help with phone numbers outside the U.S., [send your request](mailto:ptneu@microsoft.com).
+Once you have all the info, send us your request to one the following email addresses where your request falls into:
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
Once you have all the information, send us your request to one the following email addresses that is applicable to your request:
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 8]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+- **Number Porting**
+    - Porting telephone numbers **inside the U.S. or Canada**, [please send us an email at ptn@microsoft.com](mailto:ptn@microsoft.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
    - Porting telephone numbers **inside the United States or Canada**, [please send us an email at ptn@microsoft.com](mailto:ptn@microsoft.com).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 9]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+    - Porting telephone numbers **outside the U.S. or Canada**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
    - Porting telephone numbers **outside the United States or Canada**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com).
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 10]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+    
+- **Number Acquisition**
+    - Acquiring new telephone numbers for any of the following countries: **Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, [please send us an email at ptneu@microsoft.com](mailto:ptneu@microsoft.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
    - Acquiring new telephone numbers for any of the following countries: **Belgium, France, Germany, Great Britain, Ireland, Netherlands, Spain, Sweden, Switzerland**, [please send us an email at ptneu@microsoft.com].(mailto:ptneu@microsoft.com)
```

File: Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md [hunk 11]
```diff
@@ -24,14 +24,24 @@ ms.custom:
...
+    - Acquiring new telephone numbers for the **Rest of the World** , [please send us an email at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com)
```
Comment (JohanFreelancer9) [inline Teams/manage-phone-numbers-for-your-organization/contact-pstn-service-desk.md]:
```suggestion
    - Acquiring new telephone numbers for the **Rest of the World** , [please send us an email at gcstnmsd@microsoft.com](mailto:gcstnmsd@microsoft.com).
```

**Comment by ErsancMS** [comment]
Hey @JohanFreelancer9 just commited your reviews, I believe it should be good to go

**Comment by scanum** [comment]
@tonysmit 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**